cypress/e2e/inventory/cataloging/create-instance-holdings-and-item.cy.js

```diff
@@ -0,0 +1,93 @@
+import moment from 'moment';
+
+import { DevTeams, TestTypes, Permissions } from '../../../support/dictionary';
+import { INSTANCE_SOURCE_NAMES, LOCATION_NAMES } from '../../../support/constants';
+import getRandomPostfix from '../../../support/utils/stringTools';
+import TopMenu from '../../../support/fragments/topMenu';
+import InventoryInstance from '../../../support/fragments/inventory/inventoryInstance';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import Users from '../../../support/fragments/users/users';
+
+describe('Inventory', () => {
+  describe('Cataloging', () => {
+    const testData = {
+      callNumber: '331.2',
+      barcode: getRandomPostfix(),
```

Comment:
It's better use uuid() for as random barcode

Reply:
Done